index/scorch/event.go

```diff
@@ -24,6 +24,10 @@ var RegistryAsyncErrorCallbacks = map[string]func(error, string){}
 // process init()'ialization.
 var RegistryEventCallbacks = map[string]func(Event){}
 
+// RegistryContinueEventCallback should be treated as read-only after
+// process init()'ialization.
+var RegistryContinueEventCallback = map[string]func(Event) bool{}
```

Comment:
Can we consolidate this map and the above into one - it's ok to change the return type for all the existing events if you must.